### PR TITLE
Handle conversion result

### PR DESCRIPTION
**Pull request by aeyates**
This is just a Proof of Concept, so it doesn't need to be fancy. But I updated for the latest fhir2hpo lib and made it a little more readable because I might be sharing some screenshots with Peter.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,24 @@
 # Proof of Concept: HPO on FHIR
 
-This simple application can search for patients in the hardcoded sandbox and display LOINC Codes within observations along with the corresponding HPO Term if one exists. You can also see all the LOINC Codes that have been annotated and the raw annotations for each.
+This simple application can search for patients in a hardcoded sandbox and display LOINC Codes within observations along with the corresponding HPO Term if one exists. You can also see all the LOINC Codes that have been annotated and the raw annotations for each.
 
 Running the app will require the following:
 
-- Locally install the phenol library which is a dependency of the fhir2hpo library: https://github.com/monarch-initiative/phenol
-- Locally install the fhir2hpo library: https://github.com/OCTRI/fhir2hpo
+- Install Maven for building code. Run ```mvn clean install` on the following libraries to get them into your local repository. Run these in order:
```
Comment (wsams) [inline README.md]: Looks like a couple extra back ticks before `mvn clean install`.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,24 @@
...
+- The phenol library: https://github.com/monarch-initiative/phenol
+- The fhir2hpo library: https://github.com/OCTRI/fhir2hpo
+- This application
```
Comment (wsams) [inline README.md]: Is this bullet missing some text?

Comment (aeyates) [reply README.md]: No, but I can see how this is confusing. I was listing this application as a dependency of running this application. Duh! I'll remove.

Comment (wsams) [reply README.md]: haha okay I get it now.

File: src/main/resources/mustache-templates/labs.html [hunk 3]
```diff
@@ -73,6 +74,8 @@ <h2>Labs</h2>
 	function parseObservationData(data) {
 
 		$("#lab_list").html("");
+		document.getElementById('lab_list').innerHTML += 
+			"<tr><th>Observation</th><th>LOINC</th><th>Value</th><th>HPO Term with Negation</th></tr>";
```
[the remark this answers is not in the document]
Comment (aeyates) [reply src/main/resources/mustache-templates/labs.html]: That does seem better. Thanks.

**Pull request merged**